ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -107,7 +107,7 @@ Item {
                 opacity: 0.7
                 font.weight: Font.Bold
                 font.pixelSize: 51
-                color: "white"
+                color: Style.current.textColor
```

Comment:
I know it's a bit late, but this is wrong. `Style.current.textColor` is actually `black` on LightTheme. So this color is now wrong on lightTheme. This `white` is actually ok for both themes, since it is white on a circle of different colors.

Reply:
https://github.com/status-im/nim-status-client/pull/573

Reply:
Cool. Sorry yeah I just saw